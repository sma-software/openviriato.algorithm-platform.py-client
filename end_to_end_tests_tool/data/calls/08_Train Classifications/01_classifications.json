{
    "restResource": "/train-classifications",
    "httpMethod": "GET",
    "urlParameters":  null, 
    "requestBody": null,
    "CSharpMethodName": "GetTrainClassifications()",
    "CSharpMethodParameterTypes": [],
    "subcategory":  "Get Train Classifications" 
}
