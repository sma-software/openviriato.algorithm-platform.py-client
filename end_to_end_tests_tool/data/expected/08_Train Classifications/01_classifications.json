[
  {
    "description": "Unknown"
  },
  {
    "description": "Freight"
  },
  {
    "description": "Passenger"
  }
]